### DISCUSSION TITLE
Is this library alive?

### DISCUSSION THREAD

**Discussion by jonthe101** [koenbeuk/EntityFrameworkCore.Triggered#198] [General]
Hello. Just watched the demo on the dotnet community standup and this looks like a really neat library 👍
I am hesitating on using it since it's been a while since a new version was published.
Is this library still being maintained?
I see that there is a branch for .NET 8. Is it getting published anytime soon?
Thanks.

**Comment by koenbeuk** [comment]
This library is certainly alive. The v3 branch is relatively feature complete and thus far has not yet needed to update when EF Core updates. There is an experimental V4 release in the pipeline however this needs some further work as well as a clear migration path from v3 as it breaks in various ways. I've been distracted and have not yet had a chance to get that wrapped up.

**Reply by jonthe101** [reply]
Ok, sounds awesome. Thank you! I've implemented your library in our application and I really like it.
Great work 😎

**Comment by Fulgan** [comment]
I'm afraid I am having the same question about the status of this library: the idea is awesome but it doesn't seem to be maintained anymore.
The last change was more than a year ago to a branch that was never merged into master.
Would it be possible to clarify that status, please?
(And wether it's abandoned or not, thank you very much for your work!)

**Comment by PhenX** [comment]
It is alive! Updates are coming 😊
